Buildings/Fluid/HeatExchangers/BaseClasses/VariableSpeedThermalWheels/BaseClasses/Characteristics/heatExchangerEffectivenessParameters.mo

```diff
@@ -0,0 +1,27 @@
+within Buildings.Fluid.HeatExchangers.BaseClasses.VariableSpeedThermalWheels.BaseClasses.Characteristics;
+record heatExchangerEffectivenessParameters
+  "Record for heat exchanger effectiveness correction vs. wheel speed ratio"
```

Comment:
`Parameters for defining heat exchanger effectiveness at different wheel speed ratio`

Reply:
Update the model doc as suggested.